### PR TITLE
Datadog addon docs

### PR DESCRIPTION
**Pull request by khewonc**
*Issue #, if available:*

*Description of changes:*

Document the DatadogAddOn.

### CODE REVIEW

**Review by shapirov103 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Looks great overall, one small comment.

File: docs/addons/datadog.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Datadog Amazon EKS Blueprints AddOn
+
+The Datadog Blueprints AddOn deploys the Datadog Agent on Amazon EKS using the [eks-blueprints](https://github.com/aws-quickstart/cdk-eks-blueprints) [CDK](https://aws.amazon.com/cdk/) construct.
+
+## Usage
```
[the remark this answers is not in the document]
Comment (khewonc) [reply docs/addons/datadog.md:11]: Thanks for the review! I added an install section and a teammate submitted a PR for a pattern: https://github.com/aws-samples/cdk-eks-blueprints-patterns/pull/25

**Review by shapirov103 — COMMENTED** [review]
@khewonc do you mind also adding a reference to the datadog page in the mkdocs.yml (in the root) under `AddOns`? It will make it appear under the nav panel in the documentation portal and improve discoverability.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**